tortoise/contrib/pydantic/creator.py

```diff
@@ -246,10 +252,15 @@ def field_map_update(keys: tuple, is_relation=True) -> None:
         field_map_update(("pk_field",), is_relation=False)
     field_map_update(("data_fields",), is_relation=False)
     if not exclude_readonly:
-        field_map_update(
-            ("fk_fields", "o2o_fields", "m2m_fields", "backward_fk_fields", "backward_o2o_fields")
+        fs: tuple = (
```

Comment:
Please change to more verbose name for variable, right now I don't really get what `fs` means

Reply:
Fixed